### PR TITLE
Fix blog entries in newsroom, primarily

### PR DESCRIPTION
**Pull request by Scotchester**
Plus a small visual bugfix and a standardization of all of our JSON files' formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,30 +52,79 @@ And ask someone for the values to set the following environment variables:
...
+- [Bower](http://bower.io/) & [npm](https://www.npmjs.org/) for package management
+
+If you're new to Capital Framework, we encourage you to
+[start here](https://cfpb.github.io/capital-framework/).
+
+#### Installing dependencies (one time)
+
+1. Install [node.js](http://nodejs.org/) however you'd like.
+2. Install [Grunt](http://gruntjs.com/), [Bower](http://bower.io/),
+   and [Browserify](http://browserify.org/):
```
Comment (himedlooff) [inline README.md]: Do we need Browserify now?

**Comment by himedlooff** [comment]
Thank you for those JSON formatting tweaks :smile:!!!

File: README.md (hunk 1 above)
Comment (Scotchester) [inline README.md]: Mmm, probably not. Missed that when editing down the stuff I copied from the OAH README.

**Comment by Scotchester** [comment]
:grinning:

**Pull request merged**